pkgs/development/python-modules/celery-singleton/default.nix

```diff
@@ -1,4 +1,5 @@
 { lib
+, pkgs
```

Comment:
```suggestion
, redis
```

Reply:
Redis is already imported here as python here see several lines lower. we import `pkgs` to distinguish between python redis package and actual redis packages

Reply:
Importing `pkgs` is very problematic because it prevents the overrideability, I'd say pass `redisServer` from the callPackage invocation.

Reply:
Ok, got it, and it is ok to reference `pkgs` there?

Reply:
yes, at callPackage site, it's fine

Reply:
Good to know, thanks! Fixed